src/main/java/kr/co/techmoa/carracing/model/Position.java

```diff
@@ -0,0 +1,37 @@
+package kr.co.techmoa.carracing.model;
+
+import lombok.Getter;
+
+import java.util.Objects;
+
+@Getter
+public class Position {
+
+    private int coordinate;
+    private final int addon = 1;
```

Comment:
상수로 사용하면 더 좋을 것 같네요.
```suggestion
    private static final int ADDON = 1;
```